### DISCUSSION TITLE
Selected "pay the full sponsorship amount" but payment was prorated

### DISCUSSION THREAD

**Discussion by donmccurdy** [community/community#53068] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I recently started sponsoring someone, and I'm very confident I'd selected to send the full sponsorship amount ($5). Despite that, my invoice from GitHub included only the prorated amount ($0.65) for the first month.

This seems like a bug, albeit a minor one. I happened to take a screenshot before submitting the form, copied below.

<details>
<summary><i>screenshot.png</i></summary>

![github_sponsors_submission](https://user-images.githubusercontent.com/1848368/232653957-a52e412b-090c-4cec-81ea-106057cac3de.png)

</details>

**Comment by anjuan** [comment]
Hi @donmccurdy,

Thanks for writing in to let us know about this bug. We've logged it.

--Anjuan
